kotlin-testing/README.md

```diff
@@ -7,3 +7,4 @@ This module contains articles about testing in Kotlin
 - [Kotlin with Mockito](https://www.baeldung.com/kotlin/mockito)
 - [Writing Specifications with Kotlin and Spek](https://www.baeldung.com/kotlin/kotlin-spek)
 - [MockK: A Mocking Library for Kotlin](https://www.baeldung.com/kotlin/mockk)
+- [Testing Higher-Order Functions with Mockito](https://baeldung.com/kotlin/testing-higher-order-functions-with-mockito)
```

Comment:
We don't put these in at this stage, please remove it.

Reply:
Quick reminder on this one...

Reply:
Removed